commitlintplugins.test.ts

```diff
@@ -87,6 +87,38 @@ test('prefer-slash-over-backslash2', () => {
 });
 
 
+test('reject-obvious-words1', () => {
+    let commitMsgWithObviousWordAfterColon = "foo: modify bla bla";
+    let rejectObviousWord1 = runCommitLintOnMsg(commitMsgWithObviousWordAfterColon);
+    // console.log('==========>' + rejectObviousWord1.stdout)
+    expect(rejectObviousWord1.status).not.toBe(0);
+});
+
+
+test('reject-obvious-words2', () => {
+    let commitMsgWithObviousWordAfterColon = "foo: change bla bla";
+    let rejectObviousWord2 = runCommitLintOnMsg(commitMsgWithObviousWordAfterColon);
+    // console.log('==========>' + rejectObviousWord2.stdout)
+    expect(rejectObviousWord2.status).not.toBe(0);
+});
+
+
+test('reject-obvious-words3', () => {
+    let commitMsgWithoutObviousWords = "foo: bla bla bla";
+    let rejectObviousWord3 = runCommitLintOnMsg(commitMsgWithoutObviousWords);
+    // console.log('==========>' + rejectObviousWord3.stdout)
+    expect(rejectObviousWord3.status).toBe(0);
+});
+
+
+test('reject-obvious-words4', () => {
+    let commitMsgWithObviousWordsInTheMiddleOfTitle = "foo: bla update bla";
```

Comment:
@realmarv given that we cannot make a rule decide if its failure is a warning or an error, then failing on this case it's too harsh; we need to make the rule only watch out for the word if it's the first word after the area/sub-area.

Reply:
Done